kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/util/StorageConverter.java

```diff
@@ -0,0 +1,409 @@
+        }
+        print("Done with FileSystemHandler: " + clazz);
+        return fsh;
+    }
+
+    /**
+     * Read the properties file. Return null of the file is not found.
+     */
+    private static Properties readProperties(String[] args) throws IOException {
+        Properties p = new Properties(){
+
+            @Override
+            public String getProperty(String key) {
+                String prop = super.getProperty(key);
+                print("- Property " + key + "='" + prop + "'");
+                return prop;
+            }
+
+        };
+        for(int i = 0; i < args.length; i++){
+            if("-p".equals(args[i])){
+                p.load(new FileInputStream(new File(args[++i])));
+            }
+            if(args[i].startsWith("-")){
+                p.put(args[i].substring(1), args[++i]);
+            }
+        }
+        return p;
+    }
+
+    private static void printHelp(){
+        print("----------------------------------------------------------------------");
+        print("StorageConverter Help");
+        print("The StorageConverter needs properties to complete the conversion.");
+        print("These properties can either be loaded in a properties file indicated with '-p' followed by the location of the properties file");
+        print("or the properties specified in the arguments with a leading '-' followed by the values.");
+        print("");
+        print("Properties (mandatory):");
+        print("- connectionDriver: The database connection driver class.");
+        print("- connectionURL: The database connection URL.");
+        print("- connectionUsername: The database connection username.");
+        print("- connectionPassword: The database connection password.");
+        print("- sourceFileSystemHandler: This is the full class name of the source FileSystemHandler.");
+        print("- sourceFileSystemHandler.<some property>: You can set any property on the source FileSystemHandler by referensing their property names.");
+        print("- sourceBodyPath: The path set in sakai.properties for the source.");
+        print("- destinationFileSystemHandler: This is the full class name of the destination FileSystemHandler.");
+        print("- destinationFileSystemHandler.<some property>: You can set any property on the destination FileSystemHandler by referensing their property names.");
+        print("- destinationBodyPath: The path set in sakai.properties for the destination.");
+        print("");
+        print("Properties (optional):");
+        print("- deleteFromSource: Whether to delete the source files. Default false.");
+        print("- contentSql: The sql statement to retrieve the resource id's and paths. Default is new ContentServiceSqlDefault().getResourceIdAndFilePath()");
+        print("----------------------------------------------------------------------");
+    }
+
+    /**
+     * Print the text to the screen.
+     */
+    private static void print(String text){
```

Comment:
In StorageConverter.main there is a mix of using System.out.println and the use of a logger?